cmd/skaffold/app/cmd/runner.go

```diff
@@ -64,13 +64,16 @@ func createNewRunner(out io.Writer, opts config.SkaffoldOptions) (runner.Runner,
 		return nil, nil, nil, err
 	}
 
-	instrumentation.InitMeterFromConfig(configs, opts.User)
 	runner, err := v1.NewForConfig(runCtx)
 	if err != nil {
 		event.InititializationFailed(err)
 		return nil, nil, nil, fmt.Errorf("creating runner: %w", err)
 	}
-
+	instrumentation.InitMeterFromConfig(configs, opts.User)
+	_, _, err = instrumentation.InitTraceFromEnvVar()
+	if err != nil {
+		logrus.Debugf("error initializing tracing: %v", err)
+	}
```

Comment:
Why shift this after `NewForConfig`?

Reply:
This was not intentional, moved both InitMeter* and InitTrace* to be before NewForConfig